[2023-04-26T03:40:39.522+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-04-26T03:40:38.782350+00:00 [queued]>
[2023-04-26T03:40:39.528+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-04-26T03:40:38.782350+00:00 [queued]>
[2023-04-26T03:40:39.528+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T03:40:39.529+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T03:40:39.529+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T03:40:39.536+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-26 03:40:38.782350+00:00
[2023-04-26T03:40:39.541+0000] {standard_task_runner.py:55} INFO - Started process 19439 to run task
[2023-04-26T03:40:39.545+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-04-26T03:40:38.782350+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpi7ouvd7w']
[2023-04-26T03:40:39.547+0000] {standard_task_runner.py:83} INFO - Job 110: Subtask create_table
[2023-04-26T03:40:39.599+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-04-26T03:40:38.782350+00:00 [running]> on host bebb68623a17
[2023-04-26T03:40:39.648+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T03:40:38.782350+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T03:40:38.782350+00:00
[2023-04-26T03:40:39.656+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-04-26T03:40:39.660+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
                first_name varchar(255) not null,
                last_name varchar(255) not null,
                country varchar(255) not null,
                username varchar(255) not null,
                password varchar(255) not null,
                email varchar(255) not null
            );
        , parameters: None
[2023-04-26T03:40:39.662+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-04-26T03:40:39.670+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230426T034038, start_date=20230426T034039, end_date=20230426T034039
[2023-04-26T03:40:39.719+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T03:40:39.741+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
